components/BooksList/index.js

```diff
@@ -1,11 +1,13 @@
 import Book from "../Book/index.js";
 import { books } from "../../lib/books.js";
```

Comment:
Instead of importing the data to the component you probably should import it to a page and save it in a state and then pass it as a prop to all components which need the books.